File: java/link-message-client/src/main/java/link/message/client/content/NewRichMessageContent.java
package link.message.client.content;

import link.message.client.StringOrObjectSerializable;

/**
 * 新文本消息的内容
 * 
 * @author zhongt
 *
 */
public class NewRichMessageContent implements StringOrObjectSerializable {

	protected String title;
	protected String brief;
	protected String more = "点击查看全文";

	public NewRichMessageContent() {
		
	}
	
	public NewRichMessageContent(String title, String brief) {
		this.title = title;
		this.brief = brief;
	}
	
	public NewRichMessageContent(String title, String brief, String more) {
		this(title, brief);
		this.more  = more;
	}
	
	public String getTitle() {
		return title;
	}

	public void setTitle(String title) {
		this.title = title;
	}

	public String getBrief() {
		return brief;
	}

	public void setBrief(String brief) {
		this.brief = brief;
	}

	public String getMore() {
		return more;
	}

	public void setMore(String more) {
		this.more = more;
	}

}

File: net/Link.Message.Client/content/NewRichMessageContent.cs
namespace Link.Message.Client.content
{
	/// <summary>
	/// 新文本消息的内容
	/// </summary>
	public class NewRichMessageContent : IStringOrObjectSerializable
	{

	    public string Title { set; get; }
		public string Brief { set; get; }
		public string More  { set; get; }

		public NewRichMessageContent()
		{
            More = "点击查看全文";
		}

	    public NewRichMessageContent(string title, string brief): this()
		{
			this.Title = title;
			this.Brief = brief;
		}

		public NewRichMessageContent(string title, string brief, string more) : this(title, brief)
		{
			this.More = more;
		}
	}

}